packages/renderer/MdToHtml/rules/mermaid.ts

```diff
@@ -44,8 +23,6 @@ export default {
 			return self.renderToken(tokens, idx, options, env, self);
 		};
 
-		const exportButtonMarkup = isDesktop(ruleOptions.platformName) ? exportGraphButton(ruleOptions) : '';
```

Comment:
It looks like this also removes the export button in the markdown editor. It could make sense to instead hide the export button only when in the rich text editor.

Doing this may require adding a renderer option or, alternatively, updating the export button's CSS.

Thank you for working on this!